Volume/Views/BrowserView.swift

```diff
@@ -32,8 +34,82 @@ struct BrowserView: View {
         }
     }
     
+    private var navigationTitle: String {
+        switch state {
```

Comment:
Could you just make state an optional?

Reply:
It's not very declarative to manage state w/ optionals :/